webui/react/src/components/DatePickerFilter.tsx

```diff
@@ -0,0 +1,30 @@
+import { DatePicker } from 'antd';
+import { Moment } from 'moment';
```

Comment:
**suggestion:** We're using `dayjs` instead of `moment` to keep the overall build size down. `Antd` supports both and offers `dayjs` as a smaller alternative. We followed [these steps](https://ant.design/docs/react/replace-moment) to replace `moment` with `dayjs`. Docs on dayjs w/ timestamp: https://day.js.org/docs/en/timezone/timezone